front/printercartridgelevelalert.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/**
+ * ---------------------------------------------------------------------
+ *
+ * GLPI - Gestionnaire Libre de Parc Informatique
+ *
+ * http://glpi-project.org
+ *
+ * @copyright 2015-2023 Teclib' and contributors.
+ * @copyright 2003-2014 by the INDEPNET Development Team.
+ * @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ *
+ * ---------------------------------------------------------------------
+ *
+ * LICENSE
+ *
+ * This file is part of GLPI.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * ---------------------------------------------------------------------
+ */
+
+include('../inc/includes.php');
+
+Session::checkRight("printer", READ);
+Session::checkRight("cartridge", READ);
+
+Html::header(PrinterCartridgeLevelAlert::getTypeName(Session::getPluralNumber()), $_SERVER['PHP_SELF'], "tools", "printercartridgelevelalert");
```

Comment:
```suggestion
Html::header(PrinterCartridgeLevelAlert::getTypeName(Session::getPluralNumber()), $_SERVER['PHP_SELF'], "admin", "printercartridgelevelalert");
```

Reply:
Hmmm, I'm not sure why I inserted the menu item into the Administration menu. For me it should be in the tools. If so, I must change this elsewhere...

Reply:
Feature moved from Administration to a Tools sub-menu. So this suggestion is outdated.